### DISCUSSION TITLE
Group name is not showing on assigment

### DISCUSSION THREAD

**Discussion by profe-kevs** [community/community#17756] [GitHub Education]
Hi,

First issue:

I am creating a group assigment but when I create the team on GitHub Classroom (as professor) the group name is not showing when a studant accept the asiggment:

![image](https://user-images.githubusercontent.com/98577482/169335750-e52ad47a-47cd-4f98-b8e5-fbf2fce78cf7.png)

**Comment by fuhrmanator** [comment]
Hello - I think this is normal. If there are no groups created yet, the students create them (e.g. one person in each group must first create the group, then the others will see it to join it). I give instructions on what names to use for the group name, in case it's not been created yet.

**Reply by profe-kevs** [reply]
I agree, but the platform give you the option as professor to select existing set of teams:

![2022-05-19 15-57-57(1)](https://user-images.githubusercontent.com/98577482/169411598-805d1a4c-cc84-4a1e-af95-75a8967044a3.jpg)
